aries-site/src/pages/templates/navigation.mdx

```diff
@@ -1,32 +1,35 @@
-import Link from 'next/link';
-import { Anchor, Text } from 'grommet';
+import { ExampleImagePreview } from '../../components';
+import { Box } from 'grommet';
 import { CardGrid } from '../../components';
 import { getPageDetails, getRelatedContent } from '../../utils';
 
-export const title = 'Navigation';
-export const topic = 'Templates';
-export const page = getPageDetails(title);
-export const relatedContent = getRelatedContent(title);
-
 export const navPatterns = [
-  'Hub and Spoke Navigation',
-  'Persistent Navigation',
+  'Drill Down Navigation',
+  'Side to Side Navigation',
+  'Reverse Navigation',
+  'Matrix Navigation',
 ];
 export const navCards = navPatterns.map(pattern => getPageDetails(pattern));
-export const globalSidebar = getPageDetails('Global Sidebar');
-export const routerExample = getPageDetails('React Router');
-
-View navigation patterns and templates for HPE applications, learn how to incorporate Global Sidebar into applications living in the HPE as-a-service ecosystem, and see how to implement leading React routing packages with navigation templates.
 
-## Patterns
+Navigation is the overall structure that provides users screen direction to complete a task logically and easily through a website or an application. 
 
-Recommended patterns, templates, and examples for use within the context of a single application, website, or subsection within an application. Plus, demonstration for how local navigation patterns, such as [Hub & Spoke](/templates/hub-and-spoke-navigation) and [Persistent Navigation](/templates/persistent-navigation), may be combined with the [Global Sidebar](/extend/global-sidebar) for easy maneuverability across applications within the HPE ecosystem.
+Part of creating a seamless appplication or interface is creating an obvious flow from one view/interaction to the next. Think of Navigation as a story map where you help guide your user through each process. 
 
-### Navigation Within an Application
+Consider the following flow below where a user is navigated from the HPE.com home page to a Sign Up page to create an account to the view of their dashboard. 
 
-Flexible in nature, the following patterns can be applied to present contextual, supplemental, and hierachical needs within an application or subsection of a larger application.
+## Types of  Navigation
 
-- [Hub & Spoke patterns](/templates/hub-and-spoke-navigation) lend themselves well to applications primarily concerned with task execution.
-- Reach for [Persistent Navigation](/templates/persistent-navigation) when a user needs orientation or easy access to application sections.
+There are 4 different types of navigational directions a user can maneuver in:
 
 <CardGrid cards={navCards} />
```

Comment:
<img width="719" alt="Screen Shot 2021-11-18 at 12 19 21 PM" src="https://user-images.githubusercontent.com/62117454/142491397-c5883c03-21af-46a6-9742-5e22793cc176.png">

Can we add background back color where the imagery is please. 

Can we also make "Drill Down Nav" imagery smaller so it's aligned with the others please :)

Reply:
@vavalos5 in order for the SVGs to play nicely like that -- can you re-export them within the bounds of the "invisible container" like we've done in other card preview here? https://www.figma.com/file/IvYbGVlucoUMvUgbYFSG7a/Design-System-Card-Site?node-id=713%3A12

Reply:
Just confirming whether or not we were able to get replacement SVGs?